src/libtrx/game/game_flow/reader_tr1.def.c

```diff
@@ -26,7 +26,7 @@ static M_SEQUENCE_EVENT_HANDLER m_SequenceEventHandlers[] = {
     { GFS_LEVEL_COMPLETE,    nullptr, nullptr },
 
     // Events with integer arguments
-    { GFS_PLAY_LEVEL,        M_HandleIntEvent, "level_id" },
+    { GFS_LOOP_GAME,        M_HandleIntEvent, "level_id" },
```

Comment:
I think `M_HandleIntEvent` needs 1 more blank space before it

Reply:
Thanks – fixed.